crates/orchestrator/src/setup/mod.rs

```diff
@@ -43,16 +58,7 @@ pub async fn setup_cloud(setup_cmd: &SetupCmd) -> color_eyre::Result<()> {
     }
     println!("Data storage setup completed ✅");
 
-    // Queues
-    println!("Setting up queues. ⏳");
-    let queue_params = setup_cmd.validate_queue_params().expect("Failed to validate queue params");
-    match queue_params {
-        QueueValidatedArgs::AWSSQS(aws_sqs_params) => {
-            let sqs = Box::new(SqsQueue::new_with_args(aws_sqs_params, aws_config));
-            sqs.setup().await?
-        }
-    }
-    println!("Queues setup completed ✅");
+    sleep(Duration::from_secs(20)).await;
```

Comment:
can you add a comment explaining the delay with a TODO for an ideal approach?

Reply:
Valid, added.